### PR TITLE
Generate, store and load Consensus keys

### LINKED ISSUE #1208: Persist and manage Consensus keys separately from Wallet keys
**Describe what you want implemented** 
The golang process needs to generate, store and load the consensus keys

### PR DESCRIPTION
**Pull request by goshawk-3**
fixes #1208

### CODE REVIEW

**Review by autholykos — APPROVED** [review]
LGTM

**Review by herr-seppia — APPROVED** [review]
LGTM

File: cmd/dusk/loadkeys.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+			if err := keys.Save(passwd, filePath); err != nil {
+				return consensuskey.Keys{}, err
+			}
+
+			log.WithField("pubkey", util.StringifyBytes(keys.BLSPubKey)).Info("create new consensus keys")
+			break
+		}
+	}
+
+	// Password is not provided on generating a new consensus keys. Then we need
```
Comment (t00ts) [inline cmd/dusk/loadkeys.go:73]: This part of the comment seems a little misleading to me.
What I understood from the code: Password appears to be provided when generating new keys right above. It is *not* provided when the file exists and we have to load it (right below)

File: pkg/config/samples/default.dusk.toml [hunk 2]
```diff
@@ -206,6 +206,9 @@ accumulatorWorkers = 4
 
 # Information for the node to send consensus transactions with
 [consensus]
+# Path to a file that stores Consensus Keys / BLS public and secret keys
+# if file does not exist, it will be created at startup.
+keysfile = "/path/consensus.keys"
```
Comment (t00ts) [inline pkg/config/samples/default.dusk.toml:211]: The default path with `/path/` in it looks weird, but I could be missing something here...

**Review by t00ts — APPROVED** [review]
Couple minor comments, but overall looks good

**Pull request merged**